### DISCUSSION TITLE
How to filter call for specific connection between 2 people? As of now any one can join video chat without any filter.

### DISCUSSION THREAD

**Discussion by AbhayJadoun** [stasel/WebRTC-iOS#127] [Q&A]
I want to create video chat between 2 specific person and other can not join until they have id.

**Comment by stasel** [comment]
Hi @AbhayJadoun 
This demo app has only basic functionality and its purpose is to help other developers to get started with WebRTC.
In order to have more users in your app you will have to implement this functionality yourself in the signaling server. You will want a similar concept to "rooms" and "users" and make sure to have a proper exchange of SDP/candidates between the correct users

Good luck!

**Comment by AbhayJadoun** [comment]
Hi,

I am new to this. Can you update signaling code for rooms and users?
It will be a great help to us.

Thanks and Regards,

*Abhay Singh | **Senior Technical Architect L1*
*Man needs his difficulties because they are necessary to enjoy success.*
*507, **Wegmans Business Park, *
*5th floor, Sector Knowledge Park III, Greater Noida – 201308*
[image: ☎]️(+91) 991-181-6959 | 📧 ***@***.***    Office:
0120-2977075
***@***.***>

***@***.***>
